Firebase/Core/FIRApp.m

```diff
@@ -176,8 +188,16 @@ + (void)configureWithName:(NSString *)name options:(FIROptions *)options {
 
     @synchronized(self) {
       if (sAllApps && sAllApps[name]) {
-        [NSException raise:kFirebaseCoreErrorDomain
-                    format:@"App named %@ has already been configured.", name];
+        // The app already exists but check to see if multiple `configure` calls are okay.
```

Comment:
Consider a function for this duplicate code.

Reply:
Broke it out into a separate function.